lib/smart_answer_flows/benefit-cap-calculator.rb

```diff
@@ -127,7 +127,7 @@ def define
 
         next_node do |response|
           if response == "none"
-            outcome :outcome_not_affected_future
+            question :housing_benefit_amount?
```

Comment:
> Given that this fix affects both the "default" and "future" flows, it might've been simpler to apply this change before introducing the "future" flow functionality.

I agree. My aim in this branch was to reuse as many commits as possible from PR #2542, which is why I included this one as it was.